anime_downloader/commands/ezdl.py

```diff
@@ -0,0 +1,198 @@
+import logging
+import os
+
+import click
+import requests_cache
+
+from anime_downloader import session, util
+from anime_downloader.__version__ import __version__
+from anime_downloader.sites import get_anime_class, ALL_ANIME_SITES
+from anime_downloader import animeinfo
+from anime_downloader.config import Config
+
+logger = logging.getLogger(__name__)
+
+echo = click.echo
+sitenames = [v[1] for v in ALL_ANIME_SITES]
+
+# NOTE: Don't put defaults here. Add them to the dict in config
+@click.command()
+@click.argument('anime_url')
+@click.option(
+    '--episodes', '-e', 'episode_range', metavar='<int>:<int>',
+    help="Range of anime you want to download in the form <start>:<end>")
+@click.option(
+    '--play', 'player', metavar='PLAYER',
+    help="Streams in the specified player")
+@click.option(
+    '--quality', '-q', type=click.Choice(['360p', '480p', '720p', '1080p']),
+    help='Specify the quality of episode. Default-720p')
+@click.option(
+    '--force-download', '-f', is_flag=True,
+    help='Force downloads even if file exists')
+@click.option(
+    '--file-format', '-ff', default='{animeinfo_anime_title}/{animeinfo_anime_title}_{provider}_{ep_no}',
+    help='Format for how the files to be downloaded be named.',
+    metavar='FORMAT STRING'
+)
+@click.option(
+    '--provider',
+    help='The anime provider (website) for search.',
+    type=click.Choice(sitenames)
+)
+@click.option(
+    '--disable-ssl',
+    is_flag=True,
+    help='Disable verifying the SSL certificate, if flag is set'
+)
+@click.option(
```

Comment:
Isnt this automated? Why is this required?

Reply:
He still wants to give the user these options citing debugging as a reason.

Reply:
It seems he wants to give the user basically the same options as a dl command. Citing debugging as the main reason. I assume @vn-ki the command you are questioning is `-c`?

Reply:
Ssl certificate? Legacy code I was spooked to remove, can definitely remove.
-c? Remnants from before I had ratio added as flag, -r 101 is more consistent from a debugging perspective so I'm OK with removing it.